tests/test_utils/test_schemaview.py

```diff
@@ -307,6 +328,35 @@ def sv_induced_slots() -> SchemaView:
     return SchemaView(schema_str)
 
 
+@pytest.fixture
+def sv_ordering_tests() -> SchemaView:
```

Comment:
bonus tests added

Reply:
Maybe also test the case where rank is set to float(inf)